lib/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -95,20 +97,26 @@ public List<String> getUserIssuedCredentials(String userId) {
     }
 
     public void deleteVcStoreItemsIfAnyExpired(String userId) {
```

Comment:
Think we can delete this method if it's not being used anywhere else? and `getItemsWithAttributeLessThanOrEqualValue` on the datastore

Reply:
Ignore me, it's still being used